extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -741,6 +762,34 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
+   * Refreshes ad slot by fetching new creative and rendering it.
```

Comment:
Mention that it leaves the remaining creative in place until the new ad promise chain resolves